### ISSUE TITLE
OpenSSL 1.1.1d broke iOS simulator build

### ISSUE THREAD

**Issue by gjasny** [openssl/openssl#9999]
Hello,

bisecting showed that commit 87bea6550ae0dda7c40937cff2e86cc2b0b09491 broke the iOS simulator build (between 1.1.1c and 1.1.1d).

Running the following on `OpenSSL_1_1_1-stable`:

```
./Configure debug-iossimulator-xcrun
make -j$(nproc)

...

Undefined symbols for architecture x86_64:
  "_aesni_cbc_encrypt", referenced from:
      _aesni_init_key in e_aes.o
      _aesni_cbc_cipher in e_aes.o
```

Tested on macOS 10.14 with Xcode 11.

Thanks,
Gregor

**Comment by gjasny** [comment]
@bernd-edlinger Could you please take a look?

**Comment by bernd-edlinger** [comment]
I guess it will be fixed by configuring with no-asm ?

**Comment by gjasny** [comment]
Yes, `no-asm` could be used as a work-around.

**Comment by mspncp** [comment]
Congrats, you’re  really hit a cool issue number @gjasny!

**Comment by bernd-edlinger** [comment]
See #10002 for a proper fix.

**Comment by bernd-edlinger** [comment]
Fix was applied to 1.1.1 branch.
